### DISCUSSION TITLE
Add own Start Button Icon

### DISCUSSION THREAD

**Discussion by wdeb** [valinet/ExplorerPatcher#2105] [Ideas]
I would like to request for the option to add your own custom start button icon. This would be very neat as you could use a .PNG selecting from a folder, that will than be display instead of the Windows Icon!

**Comment by pyrates999** [comment]
Look at Open Shell for doing that.

**Reply by wdeb** [reply]
Already looked into other options, and the option you're giving me contains malware...
So I do rather not, as well its a feature request not an alternative request...

**Reply by pyrates999** [reply]
don't know where you got the idea that open shell contains malware.  But here's the link to it: https://github.com/Open-Shell/Open-Shell-Menu
